test/parallel/test-require-extension-over-directory.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+// fixes regression from v4
+require('../common');
+const assert = require('assert');
+const fixtures = require('../common/fixtures');
+
+const fixturesRequire = require(
+  fixtures.path('module-extension-over-directory', 'inner'));
+
+assert.strictEqual(
+  fixturesRequire,
+  require(fixtures.path('module-extension-over-directory', 'inner.js')),
+  'test-require-extension-over-directory failed to import fixture' +
+  ' requirements'
```

Comment:
There is a new eslint rule in place that prevents messages without dynamic content (or it is at least planned). I do see that it makes sense to have a individual message here, so we should either explicitly accept the message here or maybe just change the `inner.js` export content to contain a property with a string, remove the message and use `deepStrictEqual`. With the latter the error message should be explicit enough.

Reply:
@BridgeAR If that rule ever comes to pass, we can always disable it with a comment for the instances where a string literal makes sense. So this can stay as is, if the proposed lint rule is the only motivation for changing it.